dialogue-clients/src/main/java/com/palantir/dialogue/clients/ChannelCache.java

```diff
@@ -75,7 +75,7 @@ static ChannelCache createEmptyCache() {
         LIVE_INSTANCES.add(newCache);
 
         int numLiveInstances = LIVE_INSTANCES.size();
-        if ((numLiveInstances > 1 && log.isInfoEnabled()) || log.isDebugEnabled()) {
+        if ((numLiveInstances > 5 && log.isInfoEnabled()) || log.isDebugEnabled()) {
```

Comment:
What's this?

Reply:
LegacyClients has its own client factory, backed by a ChannelCache -- I loosened this constraint because it's common for apps to create a couple instances already (e.g. atlas->lock + wc factory) and the additional instance would incur this logging.
It's probably safe to push up toward 10-100 since this is primarily to detect when a new instance is created on a per-operation basis, but I didn't want to change it too much in this change.